app/controllers/facility_reservations_controller.rb

```diff
@@ -172,6 +172,10 @@ def batch_update
 
     msg_hash = batch_updater.update!
 
+    OrderAssignmentMailer
```

Comment:
I'm wondering if it would make sense to pull this into the updaters rather than the controllers. Then it just takes care of everything rather than the controller needing to deal with `newly_assigned_order_details`